### DISCUSSION TITLE
Attrs Removed from Path

### DISCUSSION THREAD

**Discussion by MicahSphelele** [varabyte/kobweb#337] [Q&A]
*I have seen that from the latest version we have removed the attrs from the Path when using Svg what could be the reason?*

**Comment by bitspittle** [comment]
The Svg code is still very very new and quite experimental, so if you have a concrete example of it not working the way you'd expect or something is missing, please reply *with concrete example code*.

In short, the problem we're trying to solve is that the JB APIs for SGV elements is very incomplete. At first we were thinking about adding extension functions for everything, but it was really awkward to use. So we're experimenting creating type-safe APIs for everything instead.

Note that in 0.14.1, we released an API where all SVG composable parameters were called `scope`. That was a mistake, and they will be renamed to `attrs` in 0.14.2, to give an API that is consistent with other Compose HTML APIs. Perhaps this is all you are looking for? If you change your Kobweb version in your `libs.versions.toml` file to "0.14.2-SNAPSHOT", you should get the new "attrs" scope name.

Does that answer your question?

**Reply by bitspittle** [reply]
Note that since 0.14.2 and beyond, SVG APIs now use the `attrs` parameter and look more like official Compose HTML APIs.
